third_party/rust/adblock/v0_8/crate/Cargo.toml

```diff
@@ -12,7 +12,7 @@
 [package]
 edition = "2021"
 name = "adblock"
-version = "0.8.3"
+version = "0.8.4"
```

Comment:
Should also bump the version in `third_party/rust/adblock/v0_8/README.chromium`?